tests/acceptance/features/apiSettings/notificationSetting.feature

```diff
@@ -211,4 +210,117 @@ Feature: Notification Settings
     And user "Brian" should have "0" emails
     When user "Brian" lists all notifications
     Then the HTTP status code should be "200"
+    And the notifications should be empty
+
+
+  Scenario: disable in-app notification for Space disabled event
```

Comment:
can we modify scenario such a way that it check only space disabled  notification not available but space share invitation notification available when disabled notification for Space disabled like we did here https://github.com/owncloud/ocis/pull/10893/files